src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create conditions
+title: Create an alert condition
```

Comment:
I'd retitle this to 'Add advanced alert conditions to your alerts' or something, just to create a clear line that differentiates this doc to the one you wrote